splinterd/src/config/toml.rs

```diff
@@ -67,6 +70,20 @@ impl PartialConfigBuilder for TomlPartialConfigBuilder {
             },
         };
 
+        if let Some(version) = self.toml_config.version {
+            debug!("Toml config has version {}", version);
+            if version == TOML_VERSION {
+                ()
+            } else {
+                return Err(ConfigError::InvalidVersion(format!(
+                    "{:?}: {} must be {}",
```

Comment:
Make this the entire message (see earlier pass-through comment). This should allow the message to be less generic (than prefixing more generic text).

Reply:
Addressed, in conjunction with that earlier comment